src/components/header/header.js

```diff
@@ -39,5 +51,6 @@ Header.propTypes = {
   name: PropTypes.string.isRequired,
```

Comment:
This prop should be optional and be conditioned. In the design, I can see that we only have the logo. So I think we can show it only if we pass it down as a prop. Something like this on `L18`

```js
{name && <div className="title">{name}</div>}
```